featuretools_tsfresh_primitives/utils.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pandas as pd
+
+import featuretools_tsfresh_primitives
+
+
+def _camel_case(snake_str):
```

Comment:
Can we rename this to `_upper_camel_case` to be more specific

Reply:
I renamed it to _pascal_case. I never knew there was a difference between camel case and pascale case :)